charts/perses/templates/statefulset.yaml

```diff
@@ -64,6 +64,12 @@ spec:
           - name: data
             mountPath: {{ .Values.config.database.file.folder }}
           {{- end }}
+          {{- if .Values.datasources }}
+          {{- if not (empty .Values.datasources) }}
```

Comment:
I'm wondering why do we need this not empty? 🤔

Reply:
you are right. We don't need it! 🙈
Removed in d3a890da7dc72fe08f9042f04c372f467db8635a